### ISSUE TITLE
CBFD missing shadow with copy auxiliary buffers to RDRAM enabled

### ISSUE THREAD

**Issue by fzurita** [gonetz/GLideN64#1992]
With the option "Copy auxiliary buffers to RDRAM" enable, Conker's shadow in CBFD is broken. What is this option useful for again?

**Comment by gonetz** [comment]
Some fb effects look/work more correct with it. For example shadows in JFG intro:
![jfg](https://user-images.githubusercontent.com/8983092/51835518-41272e80-2330-11e9-8cee-70586c54408d.png)

**Comment by fzurita** [comment]
Gotcha, why does it break some effects?

**Comment by gonetz** [comment]
It may require several hours to answer this question.

**Comment by mudlord** [comment]
@fzurita Some games use the CPU basically for post processing. Like for instance JFG's softshadows.

**Comment by fzurita** [comment]
Oh, gotcha, very informative.

**Comment by mudlord** [comment]
Yeah, some N64 devs were **extremely** smart. On N64, because access to memory and pixels by CPU is so easy, they can do this and many other effects.

On PC its tons more complicated since there is no unified memory area (you got normal GPU memory, abstracted by OpenGL in this case) and normal PC emulated RDRAM. GLideN64 allocates texture buffers for framebuffers (done on N64 by a certain RDP color/depth image command I think) to a framebuffer object in OpenGL. All these framebuffer objects are rendered independantly from main emulator RDRAM. 

Modern PC games don't have this problem as everything is done on GPU anyway for graphics, and post processing and other blending operations are now done by pixel/compute shaders in them.

This option just copies these back to main RDRAM using OpenGL specific commands, but that can come with performance penalties on PC due to AGP/PCI-E bus speeds, that and other works in OGL and drivers themselves.

I could be wrong.

**Comment by gonetz** [comment]
@mudlord you are totally correct here, thanks!
